com.ibm.wala.cast.python.jython.test/pom.xml

```diff
@@ -56,6 +56,8 @@
         <configuration>
           <forkMode>once</forkMode>
           <argLine>-Dpython.import.site=false</argLine>
+          <parallel>both</parallel>
+          <threadCount>100</threadCount>
```

Comment:
Is there any way to configure the threadCount based on the # of CPUs available?  I usually try to set this to the number of CPUs available or possibly less depending on how much memory allocation is going on

Reply:
Actually, [this is the default](https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#percorethreadcount). So, `100` is actually per core. Too much?